__fixtures__/has-outdated-deps-and-external-shrinkwrap/pnpm-lock.yaml

```diff
@@ -10,7 +10,7 @@ lockfileVersion: 5
 packages:
   /is-negative/1.1.0:
     resolution:
-      integrity: sha1-8Nhjd6oVpkw0lh84rCqb4rQKEYc=
+      integrity: sha512-+iCKT4ZcvjRnjkHnQjZ8/qfciLLGD8BFKS0GNR5VjDU6jEiwh899R0GSMkaYcuTNd7fEKXb3Qib0webe6HczNw==
```

Comment:
The `is-negative@1.1.0` package doesn't exist.

```
❯ pnpm view is-negative@1.1.0
npm ERR! code E404
npm ERR! 404 No match found for version 1.1.0
npm ERR! 404 
npm ERR! 404  'is-negative@1.1.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

I'm guessing the `sha1-8Nhjd6oVpkw0lh84rCqb4rQKEYc=` checksum was manually copied from `is-negative@2.1.0`, so I just used the sha512 for that version here.